src/components/card/CardSidebar.vue

```diff
@@ -464,6 +464,9 @@ export default {
 			this.descriptionSaving = false
 		},
 		updateTitle(newTitle) {
+			if (newTitle === '' || newTitle === ' ') {
+				return
```

Comment:
Would be nice to have some error handling, though I'm not sure the sidebar currently supports this, so maybe just show a toast with an error message.